source/_components/binary_sensor.template.markdown

```diff
@@ -66,15 +77,6 @@ sensors:
           description: Defines a template for the entity picture of the sensor.
           required: false
           type: template
-        attribute_templates:
```

Comment:
Why has this been removed?

Reply:
Yeah, the attribute_template option was added to the code after I had submitted my original (joint) pull request 4 weeks ago. I'll add these lines back in.